.circleci/config.yml

```diff
@@ -18,6 +18,16 @@ executors:
     resource_class: medium
     working_directory: ~/project
     environment:
+      architecture: "amd64"
+      GRADLE_OPTS: -Dorg.gradle.daemon=false -Dorg.gradle.parallel=true -Dorg.gradle.workers.max=2
+
+  besu_arm64_executor_med: # 2cpu, 8G ram
+    machine:
+      image: ubuntu-2004:202101-01
```

Comment:
could we remove the tag `202101-01` to get always the updated version?

Reply:
The circle-ci docs imply we need to be specific about the image tag:
https://circleci.com/docs/2.0/arm-resources/#overview